readme.md

```diff
@@ -5,30 +5,31 @@
 - Describe what `strings` and `numbers` are as types of data.
 - Get user input using `prompt()`.
 - Explain what a function is at a high level and how to pass arguments into it.
+- Explain what jQuery is and what problems it solves
 - Load jQuery into HTML
-- Leverage jQuery to:
+- Use jQuery to:
   - Select elements in jQuery.
   - Change text in HTML using jQuery.
   - Change css properties of selected elements.
-  - Add a simple event listener.
+  - Add an event listener.
 
 ## Framing
-In the previous classes we went over HTML and CSS. Content and styling. Is that enough? How can we start to think about how things are happening with interactive web pages. Let's take a look at this browser [game](http://orteil.dashnet.org/cookieclicker/) ....
+In the previous classes we went over HTML and CSS. Content and styling. Is that enough? How can we start to think about how things are happening with interactive web pages. Let's take a look at this browser [game](http://jshawl.com/fsm/) ....
 
-## Think-Pair-Share: Identify Javascript features in Cookie Clicker.
+## Think-Pair-Share: Identify Javascript features in Super Mario
```

Comment:
Add a link to the game in here. It's a little hidden at the end of the last section.